tests/integration/models/reddit/test_subreddit.py

```diff
@@ -954,6 +954,16 @@ def test_comments__with_skip_existing(self, reddit):
         # that there are at least 400 comments in the stream.
         assert count < 400
 
+    def test_comments__with_continue_after_id(self, reddit):
```

Comment:
nit: These test methods are sorted alphabetically. Could you move this to line 929?

Reply:
Done